react-hot-button-component/src/hot-button.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+class HotButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { count: 0 };
+    this.numberClick = this.numberClick.bind(this);
+  }
+
+  numberClick() {
+    this.setState({ count: this.state.count + 1 });
+  }
+
+  render() {
+    let className = 'white';
+    const count = this.state.count;
+    if (count >= 3 && count < 6) {
```

Comment:
Well done sir, the one thing I would do here is move the logic for deciding on the classname to a separate method, then call that method and assign the return value to your `className` variable. This will clean up the render method nicely and organize the code nicely!